client/src/js/jobs/components/Item/__tests__/Action.test.js

```diff
@@ -0,0 +1,48 @@
+import { JobAction } from "../Action";
+import { JobItem } from "../Item";
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['JobItem' is defined but never used. (no-unused-vars)](https://app.codacy.com/app/virtool/virtool/pullRequest?prid=4693803)